app/controllers/staff/proposal_reviews_controller.rb

```diff
@@ -30,7 +30,7 @@ def show
     rating.touch unless rating.new_record?
 
     current_user.notifications.mark_as_read_for_proposal(request.url)
-    track_and_format_edit = current_user.program_team_for_event?(current_event)
+    track_and_format_edit = current_user.program_team_for_event?(current_event) || current_user.reviewer_for_event?(current_event)
```

Comment:
you can actually just have this be reviewer_for_event?.  The way these checks work is they let anyone with that permission or higher in.  Thus a reviewer check allows reviewers, program team and organizers in.  A program team checks allows in program team or organizers but not reviewers or speakers.